### ISSUE TITLE
Not clear how to set up for two receive channels

### ISSUE THREAD

**Issue by skliper** [OpenC3/openc3-cosmos-cfdp#2]
Use case is there's two interfaces over which cosmos receives CFDP PDU's.  We've got it working for one, and set up the second but I suspect there's an issue with cfdp_port use within the microservice.  I expected to see that port set via the inputs to the microservice config such that two different ones would be used.  Or should the same cfdp_port be used on both?  

Can't say I follow the design yet so I'm not really sure how it all works, or is supposed to work.  Any chance there's a design diagram or similar somewhere that details the data flow on the cosmos side?

https://github.com/OpenC3/openc3-cosmos-cfdp/blob/fa4b65f1928dd54ceeeb074f021b97f2711757b0/microservices/CFDP/init.sh#L9
https://github.com/OpenC3/openc3-cosmos-cfdp/blob/fa4b65f1928dd54ceeeb074f021b97f2711757b0/microservices/CFDP/config/puma.rb#L13

Our config (note the inability to use a separate port on the two microservices):
```
VARIABLE source_entity_id_sc 23
VARIABLE source_entity_id_radio 22
VARIABLE destination_entity_id 25
VARIABLE root_path_sc /host/chan1_poll0
VARIABLE root_path_radio /host/chan0_poll0

VARIABLE cfdp_port 2905

MICROSERVICE CFDP CFDP_SC
  WORK_DIR .
  ROUTE_PREFIX /cfdp_sc
  ENV OPENC3_ROUTE_PREFIX /cfdp_sc
  ENV SECRET_KEY_BASE 324973597349867207430793759437697498769349867349674
  PORT 2905
  CMD rails s -b 0.0.0.0 -p 2905 -e production
  # MIB Options Follow -
  OPTION source_entity_id <%= source_entity_id_sc %>
  OPTION crcs_required false
  OPTION tlm_info CFS_SC CFDP_TLM_PDU_CHAN1 PDU
  OPTION destination_entity_id <%= destination_entity_id %>
  OPTION cmd_info CFS_SC CFDP_CMD_PDU_CHAN1 PDU
  OPTION root_path <%= root_path_sc %>

MICROSERVICE CFDP CFDP_RADIO
  WORK_DIR .
  ROUTE_PREFIX /cfdp_radio
  ENV OPENC3_ROUTE_PREFIX /cfdp_radio
  ENV SECRET_KEY_BASE 324973597349867207430793759437697498769349867349674
  PORT 2906
  CMD rails s -b 0.0.0.0 -p 2906 -e production
  # MIB Options Follow -
  OPTION source_entity_id <%= source_entity_id_radio %>
  OPTION crcs_required false
  OPTION tlm_info CFS_RADIO CFDP_TLM_PDU_CHAN1 PDU
  OPTION destination_entity_id <%= destination_entity_id %>
  OPTION cmd_info CFS_RADIO CFDP_CMD_PDU_CHAN1 PDU
  OPTION root_path <%= root_path_radio %>
```

**Comment by skliper** [comment]
I expected within the microservice for it to use the PORT setting instead of the variable cfdp_port.

**Comment by ryanmelt** [comment]
You are supposed to install the plugin multiple times, rather than duplicate the MICROSERVICE line.  Then cfdp_port will be unique because different plugins

**Comment by skliper** [comment]
@ryanmelt feel free to close, just confirmed I could receive a UNACK file over two channels/interfaces by installing the plugin twice.

**Comment by skliper** [comment]
@jmthomas - We recently updated to 5.14.2 and cfdp 1.0.1 and I can no longer install the plugin twice.  Seems like the system is now using the microservice folder name instead of the microservice name to determine if the microservice is unique or not?

```
Error installing plugin: DEFAULT: openc3-cosmos-cfdp-0.0.1.gem
DEFAULT__openc3_targets:CFDP already exists at create
```

Starting the first time we use `MICROSERVICE CFDP CFDP_SC` and the second time `MICROSERVICE CFDP CFDP_RADIO`.  Was this intended or possibly user error?  Worked back on maybe 5.12 or so, and they would show up as CFDP_SC and CFDP_RADIO on the Administrator Console.

**Comment by ryanmelt** [comment]
Installing multiple of the same plugin was fixed in 5.15.
